### PR TITLE
bug: Fix import issue in setup

### PR DESCRIPTION
**Pull request by taranjeet**

### CODE REVIEW

**Comment by taranjeet** [comment]
@deshraj : reverting your change for now, getting following error when building the package:
```
Traceback (most recent call last):
  File "/Users/tj/myapps/github/embedchain-org/embedchain-public/pyenv/lib/python3.11/site-packages/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
    main()
  File "/Users/tj/myapps/github/embedchain-org/embedchain-public/pyenv/lib/python3.11/site-packages/pyproject_hooks/_in_process/_in_process.py", line 335, in main
    json_out['return_val'] = hook(**hook_input['kwargs'])
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tj/myapps/github/embedchain-org/embedchain-public/pyenv/lib/python3.11/site-packages/pyproject_hooks/_in_process/_in_process.py", line 287, in get_requires_for_build_sdist
    return hook(config_settings)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/folders/z9/g8l9ykf51mnbqs8b5nd3lvq40000gn/T/build-env-d2vmkoad/lib/python3.11/site-packages/setuptools/build_meta.py", line 344, in get_requires_for_build_sdist
    return self._get_build_requires(config_settings, requirements=[])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/folders/z9/g8l9ykf51mnbqs8b5nd3lvq40000gn/T/build-env-d2vmkoad/lib/python3.11/site-packages/setuptools/build_meta.py", line 323, in _get_build_requires
    self.run_setup()
  File "/private/var/folders/z9/g8l9ykf51mnbqs8b5nd3lvq40000gn/T/build-env-d2vmkoad/lib/python3.11/site-packages/setuptools/build_meta.py", line 338, in run_setup
    exec(code, locals())
  File "<string>", line 3, in <module>
ModuleNotFoundError: No module named 'embedchain'

```

File: setup.py [hunk 1]
```diff
@@ -1,14 +1,12 @@
 import setuptools
 
-import embedchain.version
```
Comment (chilkaditya) [inline setup.py:3]: from embedchain import version

Comment (taranjeet) [reply setup.py:3]: hi @chilkaditya thats a good catch. Do you want to create a PR to fix this?

Comment (amjadraza) [reply setup.py:3]: This is fixed in #280. 

```
import embedchain as ec
print(ec.__version__)

```

**Pull request merged**